lib/core-integration/src/Test/Integration/Scenario/API/Byron/Transactions.hs

```diff
@@ -493,7 +493,7 @@ spec = describe "BYRON_TRANSACTIONS" $ do
         r <- request @([ApiTransaction n]) ctx link Default Empty
         verify r
             [ expectResponseCode HTTP.status200
-            , expectListSize 1 -- Now funded through a tx in the cluster setup
+            , expectListSize 10
```

Comment:
In the wrong commit - related to refactoring, not to the addition of babbage support